benchmark-service/benchmark/static/css/autocomplete.css

```diff
@@ -0,0 +1,189 @@
+#autoComplete_wrapper {
```

Comment:
You've probably noticed that the list has a little unnecessary space on the left hand side. This is because `ul` elements, by default, add a little bit of left padding. This creates space for the bullets. Since we're not using bullets (you have list-style:none on the list items (li's)), lets' nix the left padding with:
```
ul.file_list {
    padding-left: 0;
}
```